pkg/scheduler/redis_scheduler.go

```diff
@@ -25,6 +27,10 @@ const SchedKey = "scheduling"
 // pollInterval is the time interval between 2 redis polling
 const pollInterval = 1 * time.Second
 
+// eventLoopSize is the number of goroutines handling @events and triggering
+// jobs.
+const eventLoopSize = 50
```

Comment:
OK, we should make proportional to `runtime.NumCPU()` (https://golang.org/pkg/runtime/#NumCPU), maybe 4x it.